behave_test_automation/pah_behave/features/steps/steps.py

```diff
@@ -0,0 +1,10 @@
+import time
+from behave import given, then, when
+import features.steps.selectors as selector
+
+
+@when(u'User enters "{text}" into "{label}" field')
+def step_impl(context, text, label):
```

Comment:
ditto - meaningful name

Reply:
As mentioned above - this step was not a part of `login-testcase.feature`, and will be refactored in next PR